.github/scripts/run-integration-tests-group1.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+SETUP_ENVTEST_VER=${SETUP_ENVTEST_VER-v0.0.0-20221214170741-69f093833822}
+ENVTEST_K8S_VERSION=${ENVTEST_K8S_VERSION-1.25}
```

Comment:
nit: could we perhaps take the opportunity to default to more recent versions of these tools?